tests/bsim/bluetooth/host/adv/chain/src/main.c

```diff
@@ -31,11 +31,15 @@
 		bs_trace_info_time(1, __VA_ARGS__);	\
 	} while (0)
 
-#define NAME_LEN 30
+#define NAME_LEN sizeof(CONFIG_BT_DEVICE_NAME)
```

Comment:
Can't this be reverted back to how it was? Or is this change necessary?

Reply:
Ditto, separate PR